src/Bar.js

```diff
@@ -25,7 +25,11 @@ class Bar {
       dataColors: colors,
       fontFamily: 'xkcd',
       strokeColor: 'black',
+      strokeColorTitle: 'grey',
       backgroundColor: 'white',
+      isShorterLabelActive: false,
+      charNum: 5,
+      shorterLabels: [],
```

Comment:
Let's focus on `Adding negative values on bar chart` for this PR first and not changing the options for bar chart.

Because: 
1. we need to make sure these options not only valid for bar but also other charts
2.  For shorter labels, adding 3 more options seems too overwhelming. We can discuss this in another issue or PR